gcloud-java-dns/src/main/java/com/google/cloud/dns/DnsImpl.java

```diff
@@ -110,21 +110,23 @@
     dnsRpc = options.rpc();
   }
 
+  static Function<ManagedZone, Zone> zoneFromPb(final DnsOptions options) {
+    return new Function<ManagedZone, Zone>() {
+      @Override
+      public Zone apply(
+          com.google.api.services.dns.model.ManagedZone zonePb) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.google.api.services.dns.model.ManagedZone' due to existing import 'com.google.api.services.dns.model.ManagedZone'](https://www.codacy.com/app/mziccard/gcloud-java/file/2426833784/issues/source?bid=3123103&fileBranchId=3240468#l117)